backend/Models/UserAuthenticationRequest.cs

```diff
@@ -0,0 +1,14 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace CbtBackend.Models;
+
+public class UserAuthenticationRequest {
+    [NotNull]
+    public string? Email { get; set; }
+
+    [NotNull]
+    public string? Password { get; set; }
+
+    [NotNull]
+    public string[]? Scopes { get; set; }
+}
```

Comment:
We can use records for immutable body request types
```suggestion
public record UserAuthenticationRequest(string Email, string Password, string[] Scopes);
```

Reply:
personally, i don't like this because then the file contains only a single line